### PR TITLE
Feature/custom logo in public form

### PR DESCRIPTION
**Pull request by Abhinegi2**
closes: #2626

### Visible/Frontend Changes

- [x] Added logo in public form

### CODE REVIEW

**Comment by sleidig** [comment]
@george-neha , can you try to test this in the automatic deployment from the comment above (i.e. look at the issue linked in the first post to understand the requirement --> log out and log in as demo-admin + edit the existing form's image + check the public form via the link on the dashboard to see the result)

**Comment by george-neha** [comment]
There is currently no form image available in the configuration screen. 
However, I can upload a profile image, but this does not fulfill the requirement of positioning it between the title and the description. I can configure the image to be uploaded anywhere.
image

image





Circular and square images fit well in the designated space.


image
image

With  a wide logo, unable to view the entire logo as per the requirement.

image




As a user, I should have the ability to adjust/zoom in and out of the logo to position it according to my specifications.

**Comment by sleidig** [comment]
I notice that you tested on dev.aam-digital.net - however, we do not have this Pull Request (PR) here deployed on that dev server (the version running there is the one from PR #2682, which doesn't include the changes discussed here).

Sorry for the confusion, @george-neha .  You can try testing from the demo deployment of this PR (see https://github.com/Aam-Digital/ndb-core/pull/2706#issuecomment-2522879945 ). Although this will not save changes across reloads, it should allow you to try the new image configuration.

**Comment by george-neha** [comment]
Sorry for the confusion.
Uploaded a square image, and the image is distorted (see below)
image
image
Uploaded a circular/round image, and the image is distorted (see below)
image

Upload a wide logo, only this image is not too distorted.
image

**Comment by Abhinegi2** [comment]
> Sorry for the confusion. Uploaded a square image, and the image is distorted (see below) > Uploaded a circular/round image, and the image is distorted (see below)
> Upload a wide logo, only this image is not too distorted.

@george-neha I updated some rendering logic, Can you test it now?

**Comment by george-neha** [comment]
@Abhinegi2   I tested this, and the images no longer get distorted, regardless of the shape of the logo.

File: src/app/features/public-form/public-form.component.html [hunk 1]
```diff
@@ -1,7 +1,16 @@
...
   <mat-card-header>
-    <mat-card-title>{{ formConfig?.title }}</mat-card-title>
-    <mat-card-subtitle>{{ formConfig?.description }}</mat-card-subtitle>
+    <div>
+      <mat-card-title>{{ formConfig?.title }}</mat-card-title>
+      @if (formConfig?.logo) {
+        <app-display-img
+          [entity]="formConfig"
+          imgProperty="logo"
+          class="public-form-image"
```
Comment (sleidig) [inline src/app/features/public-form/public-form.component.html:9]: can we center this horizontally? It seems only left-aligned for me

**Review by sleidig — CHANGES_REQUESTED** [review]
code and functionality looks good overall :+1: 

Just one request for the layout (see comment)

File: src/app/features/public-form/public-form.component.html (hunk 1 above)
Comment (Abhinegi2) [reply src/app/features/public-form/public-form.component.html:9]: Do the below changes look good?
image

Comment (sleidig) [reply src/app/features/public-form/public-form.component.html]: great :+1:

**Review by sleidig — APPROVED** [review]
Good to go :1st_place_medal: :heavy_check_mark: 

_keeping this open until the base PR is merged, to provide git history of this feature separately_

**Pull request merged**